packages/cysync-core/src/store/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from './store';
 export * from './wallet';
 export * from './lang';
+export * from './addAccountSlice';
```

Comment:
No need to create a global state for `addAccount.strings`, `lang` slice should be able to provide this data.

Reply:
done